cdbot/cogs/maths.py

```diff
@@ -200,7 +201,24 @@ async def latex_render(self, ctx: Context, expression: str):
                 image_bytes = BytesIO()
                 image.save(image_bytes, format="PNG")
                 image_bytes.seek(0)
-                await ctx.send(file=File(image_bytes, filename="result.png"))
+
+                # send the resulting image and add a cross reaction
+                message = await ctx.send(file=File(image_bytes, filename="result.png"))
+                await message.add_reaction("❌")
+
+                # checks if the person who reacted was the original latex author and that they reacted with a cross
+                def should_delete(reaction: Reaction, user: Member):
+                    return ctx.message.author == user and reaction.emoji == "❌"
```

Comment:
```suggestion
                    return ctx.message.author == user and reaction.emoji == "🗑️"
```

Reply:
agreed

Reply:
Looks like committing only changed one of the emojis... I'll change them all then